src/Resources/snippet/storefront/paypal.en-GB.json

```diff
@@ -79,6 +79,11 @@
             "clear": "Remove card",
             "create": "Remember card for future purchases"
           }
+        },
+        "e-invoice":{
+          "paymentMethod": "Paypal - \"%paymentMethod%\"",
```

Comment:
I would not mark this as `PayPal`, as this is usually not exposed to the customer, that e.g. card payments or iDEAL is powered by PayPal.